### PR TITLE
Fix link to OWNERS path for kube-scheduler-simulator

### LINKED ISSUE #6309: fix OWNERS path for kube-scheduler-simulator in sigs.yaml
### Describe the issue

the correct URL is `https://raw.githubusercontent.com/kubernetes-sigs/kube-scheduler-simulator/master/OWNERS`

currently the sigs.yaml says `https://raw.githubusercontent.com/kubernetes-sigs/kube-scheduler-simulator/main/OWNERS`

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #6309

### CODE REVIEW

**Comment by jdumars** [comment]
Hi @championshuttler to make this change, you should not manually edit sig-list.md or README.md files inside the sig folders. Instead make your changes to sigs.yaml, then run `make`, and then commit your changes to sigs.yaml and any generated docs. The process is here: https://github.com/kubernetes/community/tree/master/generator

Let me know if you run into any problems.

File: sigs.yaml [hunk 1]
```diff
@@ -2193,7 +2193,7 @@ sigs:
     - https://raw.githubusercontent.com/kubernetes-sigs/kube-batch/master/OWNERS
   - name: kube-scheduler-simulator
     owners:
-    - https://raw.githubusercontent.com/kubernetes-sigs/kube-scheduler-simulator/main/OWNERS
+    - https://raw.githubusercontent.com/kubernetes-sigs/kube-scheduler-simulator/master/OWNERS
```
Comment (championshuttler) [inline sigs.yaml:2196]: Done 💯

Comment (jdumars) [reply sigs.yaml:2196]: Awesome! Thanks

**Pull request merged**